lib/ddtrace/contrib/grape/endpoint.rb

```diff
@@ -63,7 +63,16 @@ def self.endpoint_run(name, start, finish, id, payload)
 
           begin
             # collect endpoint details
-            api_view = payload[:endpoint].options[:for].to_s
+            api = payload[:endpoint].options[:for]
```

Comment:
@delner Is this change intended for this PR?

Reply:
@frsantos No it isn't. It was based on another PR which fixed the broken CI build. I have since rebased it back on master, so it should disappear.